### PR TITLE
Trivial: gitignore contents of db4 folder

### PR DESCRIPTION
**Pull request by murtyjones**
A lot of stuff is generated by running `./contrib/install_db4.sh .`. I had to ignore it to avoid committing to my repo and I'm sure many others must do the same.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -116,3 +116,6 @@ test/cache/*
 
 libbitcoinconsensus.pc
 contrib/devtools/split-debug.sh
+
+# Output from running db4 installation
+db4/*
```
Comment (practicalswift) [inline .gitignore]: Nit: `db4/`

**Comment by practicalswift** [comment]
utACK d3881b6f435daace615de1dfe56d4aade882b4d6 modulo nit

File: .gitignore (hunk 1 above)
Comment (murtyjones) [reply .gitignore]: Done

**Comment by scravy** [comment]
> I slipped in another one for mac users of Intellij – the `.idea` folder.

Please don't. Read the developer notes: https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#ignoring-ideeditor-files

> In closed-source environments in which everyone uses the same IDE it is common to add temporary files it produces to the project-wide .gitignore file.
> 
> However, in open source software such as Bitcoin Core, where everyone uses their own editors/IDE/tools, it is less common. Only you know what files your editor produces and this may change from version to version. The canonical way to do this is thus to create your local gitignore.

**Comment by murtyjones** [comment]
OK, it's removed.

**Pull request closed without merging**